app/experimenter/docs/adrs/0005-doc-hub.md

```diff
@@ -0,0 +1,231 @@
+# Use Docusaurus + GH Pages for the Nimbus User Doc Hub
+
+- Deciders: Lauren Zugai, Kate Hudson, and Jody Heavener, with additional input from those involved in Nimbus
+- Date: 2021-02-25
+
+Technical Story: [Documentation Hub for Nimbus Users](https://jira.mozilla.com/browse/EXP-860)
+
+## Context and Problem Statement
+
+In scaling Nimbus for 2021 and beyond, we need to provide experiment owners, data scientists, and desktop and Fenix engineers documentation to better streamline working with the platform. Following the same line of thought in [`nimbus-shared` ADR 0001](https://github.com/mozilla/nimbus-shared/blob/main/docs/adr/0001-one-console-app.md), users should expect a single source of truth for their documentation needs.
+
+The documentation that will be part of the hub includes:
+
+- A new front-page or home-page doc, likely written in MD, with a brief overview of Nimbus and instructions for users on the workflow for doc maintenance
+- New docs, likely written in MD, and depending on the chosen option, possibly optionally with React components (if Docusaurus or Storybook) or custom JS (if Docusaurus, Storybook, or `mdBook`)
+  - For Fenix engineers: SDK docs for integration in Fenix
+  - For experiment owners: How to use the console, either living here or in Mana
+  - For data scientists: How to add Jetstream configs
+  - How to read automated analysis, either living here or in Mana
+  - In the future, iOS and Gecko integration docs
+- Existing docs
+  - For Desktop engineers: [SDK docs for integration in desktop front end](https://www.notion.so/k88hudson/Nimbus-Migration-Guide-for-Desktop-Front-End-Experiments-d36c21e505f84550aad1202897fc4ba3)
+- Various Google docs and Mana links
+- Possibly existing MD docs living in other repos
+
+## Decision Drivers
+
+- Docs should be kept in a central location and cross-linking should be easy
+- Finding relevant documentation should be painless, quick, and clear for all of our users, and cover their specific doc needs, like copying a code snippet in various languages
+- Docs should be easily editable by anyone; maintenance and workflow should be straightforward and scalability should be taken into account
+- Set up, hosting, and deployment efforts should be minimal
+
+## Considered Options
+
+- Mana / Confluence
+- Docusaurus + GH Pages
+- Experimenter repo Wiki
+- Storybook
+- mdBook + GH Pages
+
+## Decision Outcome
+
+Chosen option: "Docusaurus + GH pages", because Docusaurus provides a flexible and easily updated sidebar, allows MD and custom JS when needed, and provides a robust search bar. GitHub Pages takes care of deployment efforts.
```

Comment:
You can you record the context here that we will try to keep customization to a minimum in order for this to be as low maintenance as possible?

Reply:
Cool, I'll expound upon this section a bit more to capture conversations held here. 😁